bin/ld.gold

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+if [[ "$CC_CLUSTER" == "narval" ]]; then
```

Comment:
Do we know what version(s) of Lustre are affected ? Should we make the condition based on the version of the Lustre clients ?